ci/lint.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16
+    tag: latest
```

Comment:
i guess we need to fix the go version wherever possible, specifically for the linter . As if you tag it with latest, then some linters in golangcilint won't be forward compatible with unexpected version update and we will start experiencing unexpected   lint issues in the pipeline.